src/Music/Score/Color.hs

```diff
@@ -79,7 +79,7 @@ instance HasColor a => HasColor (PartT n a)
 
 instance HasColor a => HasColor (TieT a)
 
-newtype ColorT a = ColorT {getColorT :: Couple (Option (Last (Colour Double))) a}
```

Comment:
We can also replace all uses of `Semigroup.Option (First A)` with `Maybe.First A`, and similarly for `Last`.